suite-native/notifications/src/components/ToastNotification.tsx

```diff
@@ -46,54 +47,84 @@ const IconContainerStyle = prepareNativeStyle<{
     ),
 }));
 
-const ToastNotificationVariantToStyleMap = {
+const ToastTypeToContentMap = {
+    'copy-to-clipboard': {
+        variant: 'default',
+        iconName: 'copy',
+        defaultMessage: 'Copied to clipboard',
+    },
+    'graph-error': {
+        variant: 'warning',
+        iconName: 'warningCircle',
+        defaultMessage: 'Error loading the graph',
+    },
+    'redirect-error': {
+        variant: 'error',
+        iconName: 'warningTriangle',
+        defaultMessage: 'Unable to open the link',
+    },
+} as const satisfies Record<
+    NativeToastType,
+    { variant: ToastNotificationVariant; iconName: IconName; defaultMessage: string }
+>;
+// todo change to toastNative tipe
```

Comment:
nit: I would consider creating constants file for this one as it might grow quite a bit in the future and doesn't need to be defined inside component. Resolve if you disagree

Reply:
What about leaving it there and moving it away as it grows larger?